posthog/api/hog_function.py

```diff
@@ -188,7 +188,7 @@ def to_internal_value(self, data):
                     raise serializers.ValidationError(
                         {"template_id": "Transformation functions must be created from a template."}
                     )
-        elif not has_addon:
+        elif not has_addon and data["type"] != "internal_destination":
```

Comment:
Errr nope. This should be on the line below.

If they don't have the data pipelines addon, we should _only_ allow bypassing the `template.free` part. The rest should remain in place.